packages/rancher-sachet/charts/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+annotations:
+  catalog.cattle.io/hidden: "true"
+  catalog.rancher.io/certified: rancher
+  catalog.rancher.io/release-name: rancher-sachet
```

Comment:
I think we copied this from PushProx, but I have recently been informed that catalog.rancher.io is not supported 😅 https://github.com/rancher/rancher/issues/31927#issuecomment-816995094

I plan to make the change to remove this from any other Monitoring-related charts too that have it, but we should remove this here

Reply:
fixed.